model.py

```diff
@@ -2728,6 +2731,7 @@ class Edition(Base):
     AUDIO_MEDIUM = u"Audio"
     MUSIC_MEDIUM = u"Music"
     VIDEO_MEDIUM = u"Video"
+    IMAGE_MEDIUM = u"Image"
```

Comment:
What's the goal of this? These constants are intended to go into `Edition.medium`. That implies a situation where a single image shows up in a catalog as a 'book' which may be 'borrowed' and downloaded for local reference. This isn't impossible, especially in an academic or archive context, but I don't think that's what Odilo offers.

Reply:
The goal of this is exactly what you say, we offer the possibility to catalogue images individually as a whole, as if books. Then, when a patron is logged in, he or she can consult the images, but no checkout or place a hold on them. We accept some image formats such as: jpg/jpeg, png y svg , bmp and gif to catalogue images.